everest-testing/src/everest/testing/ocpp_utils/central_system.py

```diff
@@ -24,10 +25,8 @@
 
 logging.basicConfig(level=logging.debug)
 
-
 class CentralSystem:
```

Comment:
I think a couple of these members from the `CentralSystem` can be moved to the `LocalCentralSystem`.

Reply:
Looking through the code, it seems that some (example ws_server) might not be required in the abstract central system.